evaluation.py

```diff
@@ -38,7 +38,8 @@ def eval(model, loss_function, test_iter, args, ignore_index=-100) -> Any:
     mode = model.training
     model.eval()
 
-    correct = 0
+    top_k_val = 5
```

Comment:
This should either be a flag or declared up top like a constant (with corresponding notation LIKE_THIS_EXAMPLE)